packages/subgraph/schema.graphql

```diff
@@ -724,12 +784,14 @@ type MintedEvent implements Event @entity {
     to: Bytes!
     amount: BigInt!
     data: Bytes!
+    token: Bytes!
```

Comment:
I didn't see this mapped in: `packages/sdk-core/src/mapGetAllEventsQueryEvents.ts`